app/kuma-dp/cmd/run_test.go

```diff
@@ -15,13 +15,14 @@ import (
 
```

Comment:
Ok so I don't really get ordering here.
Why `"github.com/Kong/kuma/pkg/catalog"` is not merged with section of `github.com/Kong/kuma/app/kuma-dp/pkg/dataplane/envoy`? Does it respect manual separated sections or what is the reason? 🤔

Reply:
Yes, looks like it tries to respect separation that already done. I didn't find any proves in documentation. But I deleted all "\n", run again and separation became correct.